shell/platform/windows/angle_surface_manager.cc

```diff
@@ -109,34 +82,76 @@ bool AngleSurfaceManager::Initialize() {
       d3d9_display_attributes,
   };
 
+  PFNEGLGETPLATFORMDISPLAYEXTPROC eglGetPlatformDisplayEXT =
```

Comment:
Nit: missed this when the code first went in; this should be names as a variable (so, _ rather than camelCaps)